### DISCUSSION TITLE
What wallpapers were used for the README?

### DISCUSSION THREAD

**Discussion by Alex6183** [AutoDarkMode/Windows-Auto-Night-Mode#409] [Wallpaper]
Hi, this might be a usual question, but I'm curious what wallpapers have been used for the screenshots provided in the README.

**Comment by Spiritreader** [comment]
![Demo Wallpaper 1](https://user-images.githubusercontent.com/18438899/147700745-1de000f4-2c84-4f8f-8656-ffa62ccb4152.jpg)

This is the one I used for the first screenshot. I've reduced the brightness a bit and decreased the contrast compared to the original image.

Sadly I don't know what the source is anymore because it was sent to me as well, so I can't give credit to the author.

@Armin2208 should have the other wallpaper. maybe he'll post it here as well.

A quick image reverse search has them here, although I also can't find any credit.
https://wallpaperbat.com/minimalist-colorful-wallpapers

**Comment by Armin2208** [comment]
![40tgnbeo01811](https://user-images.githubusercontent.com/32982430/147702221-464cc2b0-3eab-4b42-a325-61a7eb4a6e25.png)

I also have problems to give credits, unkown source.

**Comment by Alex6183** [comment]
Much appreciated!
